src/site/pages/contribution.md

```diff
@@ -68,9 +71,9 @@ When including a partial within a partial, by default, Handlebars will pass alon
 If there is a possibility of a block nested inside another block of the same type and you want to isolate that nested block, add a new context. For example - see how the nested box is defined below with 'newcontext' added as an attribute:
 
 ```html
-{{#> card pf-c-card--modifier="pf-m-3xl pf-m-margin"}}
+{{#> card card--modifier="pf-m-3xl pf-m-margin" card--attribute='id="base-card" aria-label="Base card"'}}
```

Comment:
This is unrelated to the issue, but should we change this "card" example to another component? The modifiers used are for the "title" component, and `.pf-m-margin` doesn't exist anymore. Although I don't see another component would support nesting the component in itself. We could just remove `card--modifier` from these examples or use one of the layouts instead.

Reply:
Wellll, namespacing params resolves the issue w/nesting anyway. I'm going to remove it.

Reply:
@michael-coker maybe toolbar layout??

Reply:
Can you nest a toolbar in a toolbar? I didn't think so, but if you can, that works for me. You could nest a grid, split, or stack inside of itself.